virttest/qemu_vm.py

```diff
@@ -4001,6 +4007,8 @@ def add_nic(self, **params):
         nic.set_if_none('queues', params.get('queues', '1'))
         if params.get("enable_msix_vectors") == "yes" and int(nic.queues) > 1:
             nic.set_if_none('vectors', 2 * int(nic.queues) + 2)
+        if params.get("romfile") is not None:
+            nic.set_if_none('romfile', params['romfile'])
```

Comment:
Looks the `.set_if_value_not_none` call would be the one for this case.

Reply:
Done